NuGet.Config

```diff
@@ -29,5 +29,6 @@
     <add key="dotnet.myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
     <add key="myget.org vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="DO NOT MERGE" value="E:\code\roslyn-internal\Closed\Test\Resources\bin\Debug" />
```

Comment:
>DO NOT MERGE [](start = 14, length = 12)

This means "do not merge to master" or "do not merge at all"?

Reply:
That was a reminder to myself to delete this line. I failed.